grafana-dashboards/services-slo-history.configmap.yaml

```diff
@@ -215,6 +240,34 @@ data:
             "skipUrlSync": false,
             "sort": 0,
             "type": "query"
+          },
+          {
+            "current": {
+              "selected": true,
+              "text": "NO",
+              "value": "-- serviceslo.target,"
+            },
+            "hide": 0,
+            "includeAll": false,
+            "label": "show SLO targets",
+            "multi": false,
+            "name": "selectTargets",
+            "options": [
+              {
+                "selected": false,
+                "text": "YES",
+                "value": "serviceslo.target,"
+              },
+              {
+                "selected": true,
+                "text": "NO",
+                "value": "-- serviceslo.target,"
```

Comment:
why need to use `--` comment out instead of make it an empty string directly?

Reply:
did not find a way to set empty string in the UI... did not try very long

Reply:
that custom variable config is hard, need to ensure grafana preserve newlines during query, otherwise all sql will be commented out after this value. Another workaround is to join with previous `,`, `NO` map to `,`, `YES` map to `, serviceslo.target,`, ugly anyway...